service/test/integration2_test.go

```diff
@@ -1708,7 +1708,7 @@ func TestAcceptMulticlient(t *testing.T) {
 			},
 		})
 		if err := server.Run(); err != nil {
-			t.Fatal(err)
+			panic(err)
```

Comment:
why?

Reply:
I can revert that. I think go `vet` was complaining at some point since the Fatal is not tied to the test itself so it should not throw a `t.Fatal`.